<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/main/background.png);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="buttonPaper">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>201</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonScissor">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>280</y>
      <width>201</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonRock">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>280</y>
      <width>201</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ScoreHuman">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>41</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arcade Normal</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ScoreRobot">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>170</y>
      <width>41</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arcade Normal</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ScoreResult">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>440</y>
      <width>351</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arcade Normal</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="assets/resources.qrc"/>
 </resources>
 <connections/>
</ui>
